### PR TITLE
Fix: eslint file extension check 

### PR DESCRIPTION
**Pull request by joepegler**

### CODE REVIEW

**Review by VGabriel45 — APPROVED** [review]

File: packages/account/src/utils/index.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
-export * from "./Types";
-export * from "./Utils";
-export * from "./Constants";
+export * from "./Types.js";
+export * from "./Utils.js";
+export * from "./Constants.js";
```
Comment (AmanRaj1608) [inline packages/account/src/utils/index.ts:3]: Why is this needed

**Review by AmanRaj1608 — APPROVED** [review]

File: packages/account/src/utils/index.ts (hunk 1 above)
Comment (joepegler) [reply packages/account/src/utils/index.ts:3]: For esm builds. https://stackoverflow.com/questions/62619058/appending-js-extension-on-relative-import-statements-during-typescript-compilat/62626938#62626938

Comment (joepegler) [reply packages/account/src/utils/index.ts:3]: In the new SDK we have a more robust solution: tsc-alias
https://github.com/bcnmy/sdk/blob/main/package.json#L25

**Review by AmanRaj1608 — APPROVED** [review]
